cmd/doctor.go

```diff
@@ -0,0 +1,92 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"runtime"
+
+	"github.com/spf13/cobra"
+	vfs "github.com/twpayne/go-vfs"
+)
+
+// doctorCmd represents the doctor command
+var doctorCommand = &cobra.Command{
+	Use:   "doctor",
+	Short: "Check your system for potential problems",
+	Long: `Check your system for potential problems. Such as:
+* Does the configuration file exist
+* Does the destination directory exist
+* Does the source directory exist
+* Is Hashicorp Vault installed?
+* Is Bitwarden installed?
+* Is LastPass installed?
+`,
+	RunE: makeRunE(config.runDoctorCommand),
+}
+
+func init() {
+	rootCommand.AddCommand(doctorCommand)
+
+}
+
+func (c *Config) runDoctorCommand(fs vfs.FS, args []string) error {
+	fmt.Printf("Arch: %s\n", runtime.GOARCH)
+	fmt.Printf("OS: %s\n", runtime.GOOS)
+
+	info, err := os.Stat(configFile)
```

Comment:
Please use `fs.Stat` here and below so this command can be tested using [`vfst`](https://godoc.org/github.com/twpayne/go-vfs/vfst) eventually.

Reply:
Updated